### PR TITLE
Make DIVs to custom TAGS for better isolation

### PR DESCRIPTION
**Pull request by dparikh**

### CODE REVIEW

File: src/ui/ui.css [hunk 1]
```diff
@@ -31,11 +31,11 @@ body {
   animation: mspin-rotate 1568.63ms infinite linear
 }
 
-.swg-loading > div {
+swg-loading > div {
```
Comment (dvoytenko) [inline src/ui/ui.css]: Let's rename this `div` as well. E.g. `swg-loading-dot` or `-item`. On the other hand - totally fine contracting them in any possible way.

File: src/ui/ui.css [hunk 2]
```diff
@@ -31,11 +31,11 @@ body {
...
   animation: mspin-revrot 5332ms infinite steps(4)
 }
 
-.swg-loading > div > div {
+swg-loading > div > div {
```
Comment (dvoytenko) [inline src/ui/ui.css]: ditto

File: src/ui/ui.css [hunk 3]
```diff
@@ -19,7 +19,7 @@ body {
   margin: 0;
 }
 
-.swg-loading {
+swg-loading {
```
Comment (dvoytenko) [inline src/ui/ui.css]: Add `display: block`

File: src/ui/ui.css [hunk 4]
```diff
@@ -31,11 +31,11 @@ body {
   animation: mspin-rotate 1568.63ms infinite linear
 }
```
Comment (dvoytenko) [inline src/ui/ui.css]: Need to add `swg-container {display: block}`. This is a general rule of thumb: as soon as you replace `div` with a `swg-x`, immediately add `swg-x {display:block}`. It'd be even better to combine them all together:

```
.swg-container, .swg-loading, .swg-x, .swg-y, .swg-etc {display: block;}
```

File: src/ui/ui.css (hunk 1 above)
Comment (dparikh) [reply src/ui/ui.css]: Done

File: src/ui/ui.css (hunk 2 above)
Comment (dparikh) [reply src/ui/ui.css]: Done

File: src/ui/ui.css (hunk 3 above)
Comment (dparikh) [reply src/ui/ui.css]: Done

File: src/ui/ui.css (hunk 4 above)
Comment (dparikh) [reply src/ui/ui.css]: Done

File: src/ui/ui.css [hunk 5]
```diff
@@ -31,11 +38,11 @@ body {
...
 
-.swg-loading > div {
+swg-loading > swg-loading-animate {
```
Comment (dvoytenko) [inline src/ui/ui.css]: No longer need compound selector.

File: src/ui/ui.css [hunk 6]
```diff
@@ -31,11 +38,11 @@ body {
...
   animation: mspin-revrot 5332ms infinite steps(4)
 }
 
-.swg-loading > div > div {
+swg-loading > swg-loading-animate > swg-loading-image {
```
Comment (dvoytenko) [inline src/ui/ui.css]: Ditto: no need for compound anymore.

File: src/ui/ui.css (hunk 5 above)
Comment (dparikh) [reply src/ui/ui.css]: Done

File: src/ui/ui.css (hunk 6 above)
Comment (dparikh) [reply src/ui/ui.css]: Done

**Pull request merged**